pkg/configuration/configuration.go

```diff
@@ -11,7 +11,9 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"time"
 
+	"github.com/patrickmn/go-cache"
```

Comment:
Do we really need an external library for this?

Reply:
> Do we really need an external library for this?
It's a question of appetite I guess. We can for sure implement this ourselves but then the effort is much higher.

> Do we really need an external library for this?
I used our internal directory to look up the different modules already in use and cross-referenced it with a deep research into stability and reliability and `go-cache` was the outcome. I personally don't have a bias, I didn't know of imcache until you mentioned it. From the docs, it seems possible to replace now or even later, as it is somehow an implementation detail.